readme.md

```diff
@@ -13,12 +15,14 @@ Features
 - A lot of controls available (including Grid, ScrollViewer, ListBox, ComboBox)
 - Routed events system (compatible with WPF)
 - Windows, Mac OS X and any Linux (32-bit or 64-bit) support
+- Currently with 23 controlls more to be added
 
-![](http://gyazo.com/81e1ae92cfba8c7a1c2a98da7da75ad7.png)
+![]()

```

Comment:
I can agree to replace my screenshot with something another, but not to just remove them :)